Ical.Net/VTimeZoneInfo.cs

```diff
@@ -133,15 +133,15 @@ public virtual IList<string> TimeZoneNames
         set => Properties.Set("TZNAME", value);
     }
 
-    public virtual IDateTime DtStart
+    public virtual CalDateTime DtStart
```

Comment:
There are many of these aliases, so maybe rather address remove the the Dt... properties in a separate PR